### ISSUE TITLE
The client is extremelly slow

### ISSUE THREAD

**Issue by greyblake** [comandeo/crystal-memcached#4]
Hi! Thanks for the library!

We were implementing memcached backend for [Kiwi](https://github.com/greyblake/crystal-kiwi/pull/1) (unified interface for key-value stores) and discovered that the client by some reason is too slow.

The following program (**100 set operations**), take about **4 seconds**. I've tried another client(ruby) and it's executed immediately, so I conclude the problem on the client side.

``` crystal
require "memcached"

memcached = Memcached::Client.new

start_time = Time.now
100.times do |i|
  memcached.set(i.to_s, i.to_s)
end
duration = Time.now - start_time
puts duration.to_f
```

Thanks!

**Comment by kostya** [comment]
try to add here https://github.com/comandeo/crystal-memcached/blob/master/src/memcached/client.cr#L55, @socket.sync = false, and it should work fast and correct because flush called by hands in needed places

**Comment by comandeo** [comment]
@greyblake Thank you very much for the feedback! I've just merged corresponding PR (#5), it seems that it speeds things up. That would be great if you could check whether the performance is now acceptable from your point of view.

Unfortunately, I do not use this library for any production code anymore. Therefore, it is not in active development. However, I would be happy to help (or to try to help) if there are any issues with the library :-)

**Comment by greyblake** [comment]
Hi, @comandeo !
Yea, that fix does the job! Thanks for the quick reply:)
